react/SlideToggle/SlideToggle.less

```diff
@@ -50,7 +51,8 @@
 
 .label {
   padding: 0;
-  display: inline;
+  line-height: @grid-row-height * 3;
+  transform: none;
```

Comment:
This doesn't look like it should be using `Text`, we are using the `raw` prop to remove the padding, setting the line-height and overriding the transform. That's about everything that `Text` gives you so need to define/look for an alternative component in my opinion